README.md

```diff
@@ -121,7 +121,7 @@ Using `go install` will copy the executable into your go `bin` dir.
 To install the application with icons etc into your operating system's standard
 application location you can use the fyne utility and the "install" subcommand.
 
-    $ go get fyne.io/fyne/v2/cmd/fyne
+    $ go install fyne.io/fyne/v2/cmd/fyne@latest
```

Comment:
This doesn't work for Go versions < 1.16 and as we support Go 1.14, we can't just show a command that will only work for some. You need to do something similar to https://developer.fyne.io/started/#installing instead.

Reply:
Look on top of this command, it says:

> "To install the application with icons etc into your operating system's standard
application location you can use the fyne utility and the "install" subcommand."


So it already describes the `install` subcommand of go>1.16 versions. Do you believe new developers who want to install fyne will use versions below 1.16? I don't think so, but if you do, let's keep it as it is.

Reply:
No, it does not describe the new install command. It is the same install command just that what you have changed it to doesn’t work for older Go releases.

Yes, I believe that some users will use an older Go version because they are what’s available by default for some older LTS Linux releases.

Reply:
Notice the difference between "fyne install" and "go install" - that may be causing some confusion.